src/CqlBuilderPanel/CqlBuilderPanel.tsx

```diff
@@ -7,7 +7,10 @@ import { useFeatureFlags } from "@madie/madie-util";
 import IncludesTabSection from "./Includes/Includes";
 import useQdmElmTranslationServiceApi from "../api/useQdmElmTranslationServiceApi";
 import useFhirElmTranslationServiceApi from "../api/useFhirElmTranslationServiceApi";
-import { CqlBuilderLookup } from "../model/CqlBuilderLookup";
+import {
+  CqlBuilderAvailableLookupData,
```

Comment:
Not sure what is AvailableLookupData. will there be UnavailableData as well?

Reply:
should I change it to CqlBuilderParsedLookupData

Reply:
Yeah, that looks more appropriate.